lib/cdo/school_autocomplete.rb

```diff
@@ -1,27 +1,33 @@
 require 'singleton'
-require 'triez'
 
 class SchoolAutocomplete
   include Singleton
 
-  def initialize
-    @schools = Triez.new value_type: :object
-    School.all.each do |school|
-      key = "#{school[:name]} #{school[:city]} #{school[:zip]}".downcase
-      @schools.change_all(:suffix, key) {school}
-    end
-  end
-
   def get_matches(query, limit)
     results = []
-    if query.length >= 3
-      @schools.search_with_prefix(query.downcase, limit: limit) do |_, school|
-        results << Serializer.new(school).attributes
+    if query.length >= 4
```

Comment:
What is this magic number 4? And why do we return an empty result set if the query length is less than 4 chars?